src/components/FormFields/index.js

```diff
@@ -113,8 +188,30 @@ class FormFields extends React.Component {
                 </FormHeader>
                 <div className="row">
                     <div className="col-sm-6">
-                        <HelDateTimeField validationErrors={validationErrors['start_time']} defaultValue={values['start_time']} ref="start_time" name="start_time" label="event-starting-datetime" />
-                        <HelDateTimeField validationErrors={validationErrors['end_time']} defaultValue={values['end_time']} ref="end_time" name="end_time" label="event-ending-datetime" />
+                        <div className="row">
+                            <div className="col-xs-12 col-md-6">
+                                <HelDateTimeField validationErrors={validationErrors['start_time']} defaultValue={values['start_time']} ref="start_time" name="start_time" label="event-starting-datetime" />
```

Comment:
For some reason, here only start date (but not start time) is present when opening the form for a new event. When we add another occurrence, the second occurrence will have autopopulated date and time, but the first occurence still is without time.